### DISCUSSION TITLE
install latest version of podman

### DISCUSSION THREAD

**Discussion by weironz** [containers/podman#14064] [General]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

how to install latest version of podman ?

i use ubuntu 22.04 LTS, it have podman 3.4.4 ,but i want use podman 4.0.3
```
root@ubuntu:~# cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04 (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

```
root@ubuntu:~# apt list podman
Listing... Done
podman/jammy 3.4.4+ds1-1ubuntu1 amd64
```

**What i want:**
```
tar -zxvf podman.tar.gz 
mv podman-v4.0.3 /usr/local/bin
```
or
``` 
apt install -y podman-v4.0.3
apt install -y podman-latest
```

**Comment by xfoobar** [comment]
You can install from source code.
https://podman.io/getting-started/installation.html

**Comment by rhatdan** [comment]
This is a discussion not an upstream issue.

**Comment by baude** [comment]
can you use launchpad and request the newer version? unfortunately none of the podman maintainers are ubuntu/debian devs so we dont have rights to decide that at all.
